docs/user_guide/emummc/making_emummc.md

```diff
@@ -17,8 +17,47 @@
 
 -----
 
+### Making a nand backup
+
+
+!!! danger "Important"
+    A NAND backup is crucial. They can be used to restore the device to a working state in case of emergencies, and will be required in order to migrate to an EmuNAND setup in the near future.
+
+	Once the backup is finished, **keep it somewhere safe.** The best backup is the one you have but never need, and the worst backup is the one you need but never made. To save space, it's recommended to compress the end-result to a `.zip` file or something similar.
+
+	It's highly recommended that you use an SD card that is formatted to FAT32 and has at least 32 gigabytes of space free. This will still work on smaller cards, but it's not ideal.
+
+
+1. Enter RCM and inject the Hekate payload
```

Comment:
This part is not consistent with the rest of the page (even though this not being your fault), please change (also on sys) with an empty tip